### DISCUSSION TITLE
cadence的调度周期问题

### DISCUSSION THREAD

**Discussion by Johnite** [uber/cadence#4374] [Q&A]
我们的workflow设置的是每分钟调度，但是超时时间是2min，理论上调度周期是每分钟应该是每分钟都会调度，但是为什么在执行的时候变成了两分钟呢？
如图：

上图中很明显是2分钟执行完成一次
我们workflow中设置的是每分钟：
![截屏2021-08-16 下午6 02 29](https://user-images.githubusercontent.com/12208001/129546868-3e6fb068-3232-4689-b8c0-a59c6b7ba028.png)

这是什么原因呢？

**Comment by longquanzheng** [comment]
你的workflow 能在一分钟之内顺利完成吗？如果不能就会影响下一次的调度. 
因为你的workflow timeout是2分钟，所以应该是workflow timeout了，刚好影响了下一次的调度。如果你要确保每分钟执行，需要把workflow timeout也设置成1分钟。

**Reply by Johnite** [reply]
确实会存在个别的activity或decision task的timeout
